lib/transport/datagouvfr/authentication.ex

```diff
@@ -36,4 +47,14 @@ defmodule Transport.Datagouvfr.Authentication do
     |> put_header("accept", "application/json")
     |> AuthCode.get_token(params, headers)
   end
+
+  defmacro __using__(_) do
```

Comment:
Why?

Reply:
maybe, that might me considered as a leftover, I can just do an alias for `Authentication`